index.php

```diff
@@ -142,10 +206,14 @@
 
             <?php if (isset($result->error)) { ?>
             Fehler: <?= $result->error ?>
-            <?php } else { ?>
-            Termin gebucht!<br/>
-            Termin-Info: <?= $result->data->name ?> <?= $result->data->starts_at ?><br/>
-            <?php } ?>
+            <?php } else { 
+						if (!array_key_exists ( 'success' , $result )) {?>
```

Comment:
thanks sören, 

could you change your editor settings to use a 2-spaces indentation instead of tabs, then re-save all the files in the 2nd commit and push again ;)